### DISCUSSION TITLE
is there any complete client-server example?

### DISCUSSION THREAD

**Discussion by rozek** [logux/logux#120] [General] [answered]
Hello!

I just read through the docs and was really impressed by what Logux should be able to do.

But I could not find any complete example with a client and a server that one could use as a starting point for own experiments. What could be helpful:

- a simple counter
- a simple key-value-store
- a simple text editor

**Comment by ai** [comment, marked as answer]
> a simple counter

We had it, but I found that people need it very rarely on the production (at least to keep all CRDT metadata in the DB).

> a simple text editor

We do not support it out-of-the-box. You need to write your own custom store.

> a simple key-value-store

Todo app https://github.com/logux/examples/

**Reply by rozek** [reply]
ah,. I see - thank you very much!
